packages/react/src/ReactElement.js

```diff
@@ -290,6 +290,11 @@ export function cloneAndReplaceKey(oldElement, newKey) {
  * See https://reactjs.org/docs/react-api.html#cloneelement
  */
 export function cloneElement(element, config, children) {
+
+  if (!element) {
```

Comment:
Let's be more specific here. Only `null` and `undefined` trigger that error, right?

Reply:
Yes it’s basically a `cannot read property ‘props’ of undefined/null` problem where it tries to copy the props of the original element I think. So it should explicitly check if element is null or element is undefined?